demos/python_demos/segmentation_demo/README.md

```diff
@@ -1,12 +1,10 @@
 # Image Segmentation Python\* Demo
 
-This topic demonstrates how to run the Image Segmentation demo application, which does inference using image
-segmentation networks like FCN8.
+This topic demonstrates how to run the Image Segmentation demo application, which does inference using semantic segmentation networks.
```

Comment:
I think you should also update `demos/README.md` accordingly, since the text there is quite similar.